### DISCUSSION TITLE
Logger configuration

### DISCUSSION THREAD

**Discussion by brunoamacos** [openmdx/openmdx#53] [Q&A] [answered]
In our logfile we have a lot of warnings like this
`04-Nov-2021 17:51:59.909 WARNUNG [ajp-nio-127.0.0.1-8109-exec-279] org.openmdx.base.rest.stream.StandardRestFormatter.printValue Sys|Collection element print failure
`
This is not a problem, it occurs because our clients are on mobile devices with frequent network failures.
But it fills the logfile and makes it dificult to find the real problems.

I tried a configuration like this  in logging.properties
`org.openmdx.base.rest.stream.handlers = 14rest.org.apache.juli.FileHandler` 
just before the standard org.openmdx.handlers, but it did not work. 
Is it possitble to do something like this?

Regards, Bruno

**Comment by wfro64** [comment, marked as answer]
Unfortunately this is not possible since `StandardStreamFormatter` performs logging with `org.openmdx.kernel.log.SysLog` which delegates to a logger which is constructed as follows:

```
...
Logger logger = LoggerFactory.getLogger()
...
```
and `org.openmdx.kernel.log.LoggerFactory.getLogger()`:

```
...
public static final String STANDARD_LOGGER_NAME = "org.openmdx.kernel.log.LoggerFactory";
Logger.getLogger(STANDARD_LOGGER_NAME);
...
```
